src/DynamoPackages/Package.cs

```diff
@@ -205,7 +205,9 @@ public void EnumerateAdditionalFiles()
                 RootDirectory,
                 "*",
                 SearchOption.AllDirectories)
-                .Where(x => !x.ToLower().EndsWith(".dyf") && !x.ToLower().EndsWith(".dll") && !x.ToLower().EndsWith("pkg.json") && !x.ToLower().EndsWith(".backup"))
+                .Where(x => !x.ToLower().EndsWith(".dyf") && !x.ToLower().EndsWith(".dll") &&
+                       !x.ToLower().EndsWith("pkg.json") && !x.ToLower().EndsWith(".backup") &&
+                       !x.ToLower().Contains(@"\" + Configuration.Configurations.BackupFolderName + @"\"))
```

Comment:
I tend to think I prefer this (more readable), but that's entirely up to you:

```csharp
// Do this only once so that's easier to maintain or update.
var backupFolderName = @"\" + Configuration.Configurations.BackupFolderName + @"\";


var nonDyfDllFiles = Directory.EnumerateFiles(...)
.Where((x) =>
{
    var path = x.ToLower();

    return = !path.EndsWith(".dyf") &&
        !path.EndsWith(".dll") &&
        !path.EndsWith("pkg.json") &&
        !path.EndsWith(".backup") &&
        !path.Contains(backupFolderName);
})
```